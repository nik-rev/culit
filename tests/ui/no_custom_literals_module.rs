//! No `custom_literal` module in the crate root.
//! I expect span to be at `#[culit]` because it's a fundamental error,
//! unrelated to any integer in particular

use culit::culit;

#[culit]
fn main() {
    let a = 10km;
}
