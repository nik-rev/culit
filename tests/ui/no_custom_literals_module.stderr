error[E0433]: failed to resolve: could not find `custom_literal` in the crate root
 --> tests/ui/no_custom_literals_module.rs:7:1
  |
7 | #[culit]
  | ^^^^^^^^ could not find `custom_literal` in the crate root
  |
  = note: this error originates in the attribute macro `culit` (in Nightly builds, run with -Z macro-backtrace for more info)
